pkg/cmd/pipelineresource/list.go

```diff
@@ -17,12 +17,12 @@ package pipelineresource
 import (
 	"fmt"
 	"os"
-	"sort"
 	"strings"
 	"text/tabwriter"
 
 	"github.com/spf13/cobra"
 	"github.com/tektoncd/cli/pkg/cli"
+	pressort "github.com/tektoncd/cli/pkg/pipelineresource/sort"
```

Comment:
Why was `pressort` added back?

Reply:
@danielhelfand It had to be, the goimports was making the alias default to `pipelineresource` and `sort` was not use as the function names in sort were in use in other places. If you check `describe` or `list` commands for `pipelinerun` or `taskrun` similar format for the alias is used for the respective sort pkgs being `prsort` and `trsort`. Following the convention, I have had to move it back to `pressort` (Pipeline resource sort).